binding/tag.go

```diff
@@ -23,29 +22,66 @@ func (h *Tag) Create(r *api.Tag) (err error) {
 func (h *Tag) Get(id uint) (r *api.Tag, err error) {
 	r = &api.Tag{}
 	path := Path(api.TagRoot).Inject(Params{api.ID: id})
-	err = h.Client.Get(path, r)
+	err = h.client.Get(path, r)
 	return
 }
 
 //
 // List Tags.
 func (h *Tag) List() (list []api.Tag, err error) {
 	list = []api.Tag{}
-	err = h.Client.Get(api.TagsRoot, &list)
+	err = h.client.Get(api.TagsRoot, &list)
 	return
 }
 
 //
 // Update a Tag.
 func (h *Tag) Update(r *api.Tag) (err error) {
 	path := Path(api.TagRoot).Inject(Params{api.ID: r.ID})
-	err = h.Client.Put(path, r)
+	err = h.client.Put(path, r)
 	return
 }
 
 //
 // Delete a Tag.
 func (h *Tag) Delete(id uint) (err error) {
-	err = h.Client.Delete(Path(api.TagRoot).Inject(Params{api.ID: id}))
+	err = h.client.Delete(Path(api.TagRoot).Inject(Params{api.ID: id}))
+	return
+}
+
```

Comment:
**hint**: Coped from addon/ and generally useful enough for all binding users to be included.
Find()
Ensure()

vvv